alibi_detect/cd/tensorflow/utils.py

```diff
@@ -1,8 +1,39 @@
 from typing import Callable
 from functools import partial
+import tensorflow as tf
 
 
 def activate_train_mode_for_all_layers(model: Callable) -> Callable:
     model.trainable = False  # type: ignore
     model = partial(model, training=True)  # Note this affects batchnorm etc also
     return model
+
```

Comment:
Similar remark as for PyTorch equivalent: any reason why it shouldn't be in https://github.com/SeldonIO/alibi-detect/tree/master/alibi_detect/utils/tensorflow?